pkg/util/util.go

```diff
@@ -248,3 +250,12 @@ func ToString(v interface{}) string {
 	}
 	return s
 }
+
+// AssertError asserts on the existence of an error for tests
+func AssertError(t *testing.T, hasErr bool, err error) {
```

Comment:
I don't like pulling in the testing and assert packages for releases, that means all agents will have this package as well. Could we revert the use of this function?

Reply:
Yeah